packages/ckeditor5-list/src/listformatting/listitemfontfamilyintegration.ts

```diff
@@ -52,6 +54,14 @@ export default class ListItemFontFamilyIntegration extends Plugin {
 			isFormatting: true
 		} );
 
+		model.schema.addAttributeCheck( context => {
+			const item = context.last;
+
+			if ( !item.getAttribute( 'listItemId' ) ) {
```

Comment:
```suggestion
			if ( !item.hasAttribute( 'listItemId' ) ) {
```

Reply:
There is no hasAttribute in this context.